public/js/routes.js

```diff
@@ -266,12 +282,48 @@ angular.module( 'madisonApp' )
           authorizedRoles: [ USER_ROLES.all ]
         }
       } )
-      .state( 'content', {
-        url: '/{page:faq|about|privacy-policy|copyright|terms-and-conditions|404}',
-        controller: 'ContentController',
-        templateUrl: '/templates/pages/content.html',
+      .state( '404', {
+        url: '/404',
+        controller: '404Controller',
+        templateUrl: '/templates/pages/404.html',
```

Comment:
The main issue with that is that it's used as the `otherwise` state, so it needs special handling.